academic-calendar/react/src/components/CareerFilter.jsx

```diff
@@ -7,19 +7,18 @@ import API from '../api'
 const DropDownItems = (props) => {
   return Object.entries(props.careers).map((entry, index) => {
     return (
-      <Dropdown.Item key={index} eventKey={entry[1].code} active={entry[1].code === props.career}>{entry[1].description}</Dropdown.Item>
+      <Dropdown.Item data-testid="career-option" key={index} eventKey={entry[1].code} active={entry[1].code === props.career}>{entry[1].description}</Dropdown.Item>
     )
   })
 }
 
 const CareerFilter = (props) => {
   const [careers, setCareers] = useState([])
-
   useEffect(() => {
     if (!careers.length) {
-      API('careers')
-        .then(resp => setCareers(resp.data.careers))
-        .catch(err => console.log(err))
+      API(`careers`)
```

Comment:
Why the backticks?

Reply:
I was adding and removing things to the API function to experiment how Jest interact with it, and accidentally left it in there, same thing goes for the superfluous braces. Fixed in the latest commit.